package.json

```diff
@@ -38,18 +41,18 @@
     "lucide-react": "^0.453.0",
     "negotiator": "^0.6.3",
     "next": "^14.2.4",
-    "next-auth": "^4.24.7",
+    "next-auth": "^5.0.0-beta.25",
```

Comment:
Do we want a beta ver in prod? Pls check whether this is needed, thx

Reply:
yes, only parts are in beta, like jwt 
does not affect our use cases